### PR TITLE
Bug 1658846 - Allow users to enable and disable their webhooks

### PR DESCRIPTION
**Pull request by LissetCuevas**
@dklawren I made some little changes that I saw, and the UI is like this:
image

### CODE REVIEW

File: extensions/Webhooks/Extension.pm [hunk 1]
```diff
@@ -143,10 +146,26 @@ sub user_preferences {
...
 
+      my $dbh      = Bugzilla->dbh;
+      my $webhooks = Bugzilla::Extension::Webhooks::Webhook->match(
+        {user_id => $user->id});
+
+      $dbh->bz_start_transaction();
+      foreach my $webhook (@$webhooks) {
+        my $connector = $push->connectors->by_name('Webhook_' . $webhook->id);
+        my $config = $connector->config;
+        $config->{enabled} = trim($input->{$connector->name . ".enabled"});
```
Comment (dklawren) [inline extensions/Webhooks/Extension.pm:164]: We may need to validate the input here to make sure that the value is either 'Enabled' or 'Disabled'. Sneaky users can modify values before POST to circumvent checks to do bad things.

Comment (LissetCuevas) [reply extensions/Webhooks/Extension.pm]: I hadn't thought about sneaky users before :open_mouth:

File: extensions/Webhooks/template/en/default/hook/global/user-error-errors.html.tmpl [hunk 2]
```diff
@@ -18,4 +18,8 @@
   [% title = "Define a URL" %]
   You didn't define a URL for your webhook. Define one.
 
+[% ELSIF error == "invalid_option" %]
```
Comment (dklawren) [inline extensions/Webhooks/template/en/default/hook/global/user-error-errors.html.tmpl:21]: `webhook_invalid_option`

File: extensions/Webhooks/Extension.pm [hunk 3]
```diff
@@ -161,8 +161,13 @@ sub user_preferences {
...
         my $config = $connector->config;
-        $config->{enabled} = trim($input->{$connector->name . ".enabled"});
-        $config->update();
+        my $status = trim($input->{$connector->name . ".enabled"});
+        if ( $status eq 'Enabled' || $status eq 'Disabled' ){
+          $config->{enabled} = $status;
+          $config->update();
+        }else{
+          ThrowUserError('invalid_option');
+        }
```
Comment (dklawren) [inline extensions/Webhooks/Extension.pm:170]: good

**Pull request merged**